### ISSUE TITLE
support pre-set searches via --pattern

### ISSUE THREAD

**Issue by iloveitaly** [noborus/ov#430]
Less allows for `--pattern` to prefill searches. This is helpful for navigating git diffs, and other common formats passed to a pager. Would be awesome if this could be supported.

https://github.com/iloveitaly/dotfiles/blob/fcab1786d8cfb990c802447b77dc96f4ae3ab398/.gitconfig#L11-L13

**Comment by noborus** [comment]
thank you.
I would like to implement this as well.

**Comment by noborus** [comment]
I plan to release a version that includes the pattern option, but if you want to use it with `diff-so-fancy` like this time, it may be better to use `--section-delimiter`.
```
diff-so-fancy| ov --section-delimiter '^(Date|added|deleted|modified): '
```
Please try it if you like.

**Comment by noborus** [comment]
Implemented in v0.32.0. thank you.

**Comment by iloveitaly** [comment]
Awesome! Thank you!
